modules/standard/Assert.chpl

```diff
@@ -2,69 +2,25 @@
  * Copyright 2020 Hewlett Packard Enterprise Development LP
  * Copyright 2004-2019 Cray Inc.
  * Other additional copyright holders may be indicated within.
- * 
+ *
```

Comment:
This makes me wonder whether our copyright header will choke on this (or should have choked on the previous version?)

Reply:
ugh, I haven't realized this. Probably we have two "versions" w.r.t whitespace and I picked up the wrong one when I bring back the Assert module. I am not sure what is it that you are wondering exactly, but I'll make sure that there is no diff here for PR cleanliness

Reply:
There's a script that makes sure all key files have the right copyright header.  I'm not certain how sensitive it is to trailing whitespace or not.

Reply:
Ah I see. I'll look that up